.github/workflows/root-ci-config/build_root.py

```diff
@@ -367,13 +367,18 @@ def build(options, buildtype):
 @github_log_group("Create binary packages")
 def create_binaries(buildtype):
     os.makedirs(f"{WORKDIR}/packages/", exist_ok=True)
-    result = subprocess_with_log(f"""
-        cd '{WORKDIR}/build' && cpack -B {WORKDIR}/packages/ --verbose -C {buildtype}
-    """)
+    result = subprocess_with_log(f"cd '{WORKDIR}/build'")
+    if result != 0:
+        die(result, "Failed to cd to build dir")
 
+    result += subprocess_with_log(f"cpack -B {WORKDIR}/packages/ --verbose -C {buildtype}")
     if result != 0:
         die(result, "Failed to generate binary package")
 
+    result = subprocess_with_log(f"cd '{WORKDIR}'")
+    if result != 0:
+        die(result, "Failed to cd to work dir")
+
```

Comment:
Did you test that this actually works? `cd` executed in a subprocess should not leave an effect on the parent process...